site/pages/component/gc-features-fr.hbs

```diff
@@ -270,6 +270,67 @@
 	&lt;/div&gt;
 &lt;/section&gt;</code></pre>
 
+<h5>Cohabit design pattern</h5>
+<p>To be used in this context:</p>
+<pre><code>&lt;div class="row wb-eqht-grd mrgn-tp-xl"&gt;
+	&lt;div class="col-md-8"&gt;
+		&lt;section class="gc-features"&gt;
+			&lt;!-- gc-feature here --&gt;
+		&lt;/section&gt;
+	&lt;/div&gt;
+	&lt;div class="col-md-4"&gt;
+		&lt;!-- Cohabit component here. For example the Follow us component --&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h6>One feature tile - Cohabit design pattern</h6>
+<pre><code>&lt;div class="row wb-eqht-grd mrgn-tp-xl"&gt;
```

Comment:
On the one feature tile pattern, I don't think that we should be using equal height at all since when the cohabit section has a significant height, it makes the feature tile link's clickable area huge because of the `stretched-link` class.
```suggestion
<pre><code>&lt;div class="row mrgn-tp-xl"&gt;
```

Reply:
It is not for the gc-feature it-self, but for the grid container that have gc-feature and another feature like gc-followus

Reply:
Ok, after our discussion I see, I am removing it.